### PR TITLE
new snfeff 4.2 ; rm of useless .jar ; new snpsift

### PR DESCRIPTION
**Pull request by lecorguille**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Part of the GalaxyP hackathon
https://github.com/galaxyproteomics/tools-galaxyp/issues/57

### CODE REVIEW

File: recipes/snpeff/4.1l/snpeff.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 # snpEff executable shell script, adapted from VarScan shell script
+jar_name="snpEff.jar"
```
Comment (yhoogstrate) [inline recipes/snpeff/4.1l/snpeff.sh]: snpeff_jar_name? Otherwise it's quite generic.

Comment (lecorguille) [inline recipes/snpeff/4.1l/snpeff.sh]: That what I wanted since, this kind of wrapper is used everywhere. I'm lazy!

Comment (yhoogstrate) [inline recipes/snpeff/4.1l/snpeff.sh]: Look at all your contributions, you're not!

**Comment by bgruening** [comment]
Would be nice if we can migrate to the python based wrappers with the next release.
Thanks!
Bjoern

**Pull request merged**